sql/hive/src/test/scala/org/apache/spark/sql/hive/parquetSuites.scala

```diff
@@ -716,6 +717,39 @@ class ParquetSourceSuite extends ParquetPartitioningTest {
     }
   }
 
+  test("Alias used in converted data source tables") {
```

Comment:
Do we really need a test for it? This PR looks like a cleanup to me, and making one `Subquery` into two is not our target, making the code cleaner is.